src/main/java/duke/constant/CommandConstant.java

```diff
@@ -0,0 +1,16 @@
+package duke.constant;
+
+/**
+ * A class that contains constants related to Command.
+ */
+public final class CommandConstant {
```

Comment:
I like how you use constants for all related fields, but perhaps you can consider enums?